src/pages/Films/Films.tsx

```diff
@@ -1,45 +1,67 @@
 import { useEffect, useState } from "react";
 import * as S from "./Films.Styled";
 import Carousel from "../../components/Carousel/Carousel";
-import FilmsHeroImage from "../../components/FilmsBackground/FilmsBackground";
 import FilmsSearch from "../../components/FilmsSearch/FilmsSearch";
+import Toast from "../../components/Toast/Toast";
+import FilmsHeroContainer from "../../components/FilmsHeroImage/FilmsHeroImage";
 
-function Films() {
-	const [selectedGenre, setSelectedGenre] = useState(["Sci-Fi"]);
+function FilmsPage() {
+	const [selectedGenre, setSelectedGenre] = useState(["Popular"]);
 	const [fetchedFilms, setFetchedFilms] = useState([]);
-	const [heroImage, setHeroImage] = useState<string>("");
+	const [heroImage, setHeroImage] = useState<{
+		backdrop_path: string;
+		original_title: string;
+		overview: string;
+	} | null>(null);
```

Comment:
Typ do wyniesienia na zewnątrz, trochę zaburza nam czytelnosc, na pierwszy rzut oka ktos powie ze to default value, a to tak na prawdę obiekt z typami, poza tym mamy tutaj _ co powinno byc zamienione na camelCase (chyba ze to prosto z API leci)